src/pages/scripts/tutorials/arguments/index.md

```diff
@@ -6,7 +6,7 @@ keywords:
 ---
 # Passing Arguments
 
-Arguments/parameters can now be passed to UXP scripts. The arguments passed to the scripts can be used in the same way as any other argument. The `script.args` API can be used to access the arguments passed to the script as an array, the following sections demonstrate the various use cases.
+Since InDesign v18.4 arguments/parameters can be passed to UXP scripts. The arguments passed to the scripts can be used in the same way as any other argument. The `script.args` API can be used to access the arguments passed to the script as an array, the following sections demonstrate the various use cases.
```

Comment:
@anoopvalomkot Whats `sampleclient` in line #22? Provide a link where developers can learn more about it.

Reply:
@anoopvalomkot  line #29. Is it correct to say 
`<path_to_sampleclient>` instead of `../../../../mac/debug_cocoa64/sampleclient`  ?

Provide equivalent path for Windows.

Reply:
Missing documentation on InDesign Server makes this section out-of-context. The 'Introduction' tab should call out these details. @anoopvalomkot

Reply:
@anoopvalomkot  line # 55 says 'Unlike on InDesign Server'. As per the example, there is no difference between passing arguments in IDS or via doScript API. Please explain.

Reply:
@padmkris123 We can remove the below line from documentation, there is no difference between InDesign and IDS.
_**Unlike on IDS, `script.args` fetches the values of the arguments passed to the external script**_

Reply:
@padmkris123 IDS documentation is at https://helpx.adobe.com/in/indesign/using/indesign-server.html
We could give a link to it.

Reply:
this doc needs work!

Reply:
sample client, as I explained in another thread is a SOAP client that sends a request to IDS to execute a script. This is a C++ application packaged with IDS. Customers usually have their own implementation of the soap client.  I don't have any links to documentation on it, other than the IDS documentation that is shared above.

Reply:
Regarding the sampleClient's path on windows or mac, I don't see it captured in the IDS documentation. The path from my local installation is below, but I am not sure if it will be same in customer machine.

C:\Program Files\Adobe\Adobe InDesign Server 2023\sampleclient

yes, there is no need to refer as '../../../../mac/debug_cocoa64/sampleclient', we could mention <path_to_sampleclient>

Reply:
@padmkris123 
I have created below JIRA ticket as requested by you, feel free to add any additional input in the ticket.
https://jira.corp.adobe.com/browse/ID-4241897